src/cmd/builtin/dump.c

```diff
@@ -426,6 +497,10 @@ static int cmd_dump (optparse_t *p, int ac, char *av[])
         content_flags |= CONTENT_FLAG_CACHE_BYPASS;
         kvs_checkpoint_flags |= KVS_CHECKPOINT_FLAG_CACHE_BYPASS;
     }
+    /* default = 1, no parallelism */
+    async_max = optparse_get_int (p, "maxreqs", 1);
+    if (async_max <= 0)
+        log_err_exit ("invalid value for maxreqs");
```

Comment:
I wonder if the default should be something like 8?

It feels like a case where a little parallelism is likely to help a lot and yet not be too hard on the system, but I guess we can always raise it later.